InventoryComponents/InventoryHUD/InventoryHUD.gd

```diff
@@ -3,13 +3,18 @@ extends Control
 @onready var _gold_text : Label = $CanvasLayer/ColorRect/GoldText
 @onready var _torch_text : Label = $CanvasLayer/ColorRect/TorchText
 @onready var _relic_component : InventoryHUDRelicComponent = $CanvasLayer/ColorRect/Relics
+@onready var _debug_buttons : Control = $CanvasLayer/Buttons
+@export var _activate_debug_buttons : bool
```

Comment:
Yes, did you want something else?

Reply:
yes, did you check the DEBUG_VAR file ?
I would have wanted this to be put there, because there are associated tests to check everything is set to false for the release